src/Stratis.Bitcoin/Configuration/Settings/ConnectionManagerSettings.cs

```diff
@@ -182,6 +182,8 @@ public static void BuildDefaultConfigurationFile(StringBuilder builder, Network
             builder.AppendLine("####ConnectionManager Settings####");
             builder.AppendLine($"#The default network port to connect to. Default { network.DefaultPort }.");
             builder.AppendLine($"#port={network.DefaultPort}");
+            builder.AppendLine($"#Forces the node to accept incoming connections.");
```

Comment:
Not sure 'forces' is really the intention, it's more that it allows them when `-listen` is set.